homeassistant/components/elkm1/alarm_control_panel.py

```diff
@@ -152,7 +181,10 @@ def device_state_attributes(self):
             attrs["arm_up_state"] = ArmUpState(elmt.arm_up_state).name.lower()
         if elmt.alarm_state is not None:
             attrs["alarm_state"] = AlarmState(elmt.alarm_state).name.lower()
-        attrs["changed_by_keypad"] = self._changed_by_keypad
+        attrs[ATTR_CHANGED_BY_KEYPAD] = self._changed_by_keypad
+        attrs[ATTR_CHANGED_BY_TIME] = self._changed_by_time
+        attrs[ATTR_CHANGED_BY_ID] = self._changed_by_id
+        attrs[ATTR_CHANGED_BY] = self._changed_by
```

Comment:
Implement the property `changed_by` for this one.

https://github.com/home-assistant/core/blob/5ea4a68aae382d3486e76a474e24d0db7f59119a/homeassistant/components/alarm_control_panel/__init__.py#L110-L113